src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -449,14 +508,12 @@ export default class QuestionnaireForm extends Component {
 
   // create the questionnaire response based on the current state
   outputResponse(status) {
-    console.log(this.state.sectionLinks);
-
     var qr = this.getQuestionnaireResponse(status);
 
     if (status == "in-progress") {
-      localStorage.setItem(qr.questionnaire, JSON.stringify(qr));
-      alert("Partial QuestionnaireResponse saved");
-      console.log("Partial QuestionnaireResponse saved.");
+      this.storeQuestionnaireResponseToEhr(qr);
+      this.popupClear("Partially completed form (QuestionnaireResponse) saved to EHR", "OK", true);
```

Comment:
This popup shows up even when the store request fails.

Reply:
Fixed